desc/compute/_metric.py

```diff
@@ -1823,3 +1823,74 @@ def _gradtheta(params, transforms, profiles, data, **kwargs):
 def _gradzeta(params, transforms, profiles, data, **kwargs):
     data["|grad(zeta)|"] = jnp.sqrt(data["g^zz"])
     return data
+
+
+@register_compute_fun(
+    name="gbdrift",
+    label="\\mathrm{gradB-drift}",
+    units="~",
```

Comment:
it doesn't look like this is actually unitless? shouldn't it be 1/Tm^2 or something?

Reply:
Yes. Corrected.